index.js

```diff
@@ -18,10 +20,16 @@ const m = module.exports = opts => {
 		return Promise.reject(new Error(`Couldn't find the file ${opts.filename}`));
 	}
 
-	const run = remark().use(remarkLint, config).process;
-	const file = toVfile.readSync(readmeFile);
+	return pkgConf('awesome-lint').then(configProject => {
+		return deepAssign(config, configProject);
+	}, () => {
+		return config;
+	}).then(() => {
```

Comment:
FYI: because of how `Object.assign` works, [the call above](https://github.com/sindresorhus/awesome-lint/pull/17/files#diff-168726dbe96b3ce427e7fedce31bb0bcR24) reassigns `config` to be the new object that gets returned from this call, so I don't need use an argument here since `config` is already in scope.

I'd rather not reuse or introduce a new variable for `config`, which is I kept this as is. Let me know if you think I should change this to be more explicit.